### PR TITLE
Fixed avatar in topBar

### LINKED ISSUE #2048: Profile avatar is not displayed in toolbar
Avatar is not displayed on the top of the screen, near the login, while it is expected to be.
screenshot

### PR DESCRIPTION
**Pull request by Cheshiriks**
Closes #2048

### CODE REVIEW

**Review by nulls — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/topbar/TopBarUserField.kt [hunk 1]
```diff
@@ -97,7 +98,7 @@ private fun topBarUserField() = FC<TopBarUserFieldProps> { props ->
                             }
                         }
                     }
-                    avatar?.let { avatar ->
+                    props.userInfo?.avatar?.let {
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/topbar/TopBarUserField.kt:101]: Here you use avatar from `props`

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/topbar/TopBarUserField.kt [hunk 2]
```diff
@@ -53,7 +54,7 @@ private fun topBarUserField() = FC<TopBarUserFieldProps> { props ->
     val navigate = useNavigate()
     var isLogoutModalOpen by useState(false)
     var isAriaExpanded by useState(false)
-    val (avatar, setAvatar) = useState(props.userInfo?.avatar?.let { "/api/$v1/avatar$it" })
+    val (avatar, setAvatar) = useStateFromProps("/api/$v1/avatar${props.userInfo?.avatar}")
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/topbar/TopBarUserField.kt:57]: Here you use avatar from state

**Review by sanyavertolet — COMMENTED** [review]
Why?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/topbar/TopBarUserField.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/topbar/TopBarUserField.kt:101]: No. I use avatar from state, but state is nullable and contains the full link. here I just check that the avatar exists

Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/topbar/TopBarUserField.kt:101]: Oh sure yes, by bad

**Review by sanyavertolet — APPROVED** [review]

**Pull request merged**